### DISCUSSION TITLE
Why is Exception.TargetSite not excluded in .NET Core?

### DISCUSSION THREAD

**Discussion by rcdailey** [RehanSaeed/Serilog.Exceptions#647] [Q&A]
I noticed that when using `WithIgnoreStackTraceAndTargetSiteExceptionFilter()` in my .NET 7 console application, the `TargetSite` property is not filtered out. When I look at the code I see this:

![image](https://user-images.githubusercontent.com/1768054/206720603-e30925c5-a2a5-4a5c-a420-9e33bb709dca.png)

Why is `TargetSite` only filtered out in .NET 4.6.1 or 4.7.2? Seems like it should do what the user thinks it will do regardless of .NET version...

**Comment by RehanSaeed** [comment]
I believe it was not available in early versions of .NET Core. Now that it is, we should add it back in. Do you want to create a PR with this update?

**Reply by rcdailey** [reply]
I wish I could, but I'm up to my eyeballs in volunteer work already.
